app/index.js

```diff
@@ -291,6 +291,7 @@ module.exports = yeoman.Base.extend({
 					projectHomepage: props.projectHomepage,
 					projectRepositoryType: props.projectRepositoryType,
 					projectRepository: props.projectRepository,
+					banner: config.banner,
```

Comment:
We now have a duplicated banner property in this code block :grimacing:

See [L278](https://github.com/micromata/generator-bootstrap-kickstart/pull/39/files/f4e7ddf04983102f4b30f099aa1962b08141e728..72169d48e5d6c0245a92f38979a0b4cc689a48ab#diff-2018087f584c4398b5c3a23fc0e5f9dbL278)